aws-resiliency-yamls/aws-resiliency-service-accounts.yaml

```diff
@@ -2,6 +2,15 @@
 #
 # Note: This YAML file is just an example and cannot be used as-is.
 # You must fill it out with the appropriate information for your deployment to be able to use it.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nxrm-nexus-27385-deployment-sa
```

Comment:
We should add the following comment next to this:

`The name you specify here should be the same as the one specified to the eksctl create iamserviceaccount during the setup of the [secrets-store-csi-driver-provider-aws](https://github.com/aws/secrets-store-csi-driver-provider-aws)`

Reply:
done